src/main/java/io/github/opencubicchunks/cubicchunks/client/multiplayer/ClientCubeCache.java

```diff
@@ -160,13 +157,11 @@ public void refreshEmptySections(LevelCube chunk) {
         }
 
         public boolean inRange(int x, int y, int z) {
-            return Math.abs(x - this.viewCenterX) <= this.cubeRadius
-                && Math.abs(y - this.viewCenterY) <= this.cubeRadius
-                && Math.abs(z - this.viewCenterZ) <= this.cubeRadius;
+            return Math.abs(x - this.viewCenterX) <= this.cubeRadius && Math.abs(y - this.viewCenterY) <= this.cubeRadius
+                    && Math.abs(z - this.viewCenterZ) <= this.cubeRadius;
         }
 
-        @Nullable
-        @TransformFromMethod(owner = @Ref(ClientChunkCache.Storage.class), value = @MethodSig("getChunk(I)Lnet/minecraft/world/level/chunk/LevelChunk;"), visibility = PUBLIC)
+        @Nullable @TransformFromMethod(owner = @Ref(ClientChunkCache.Storage.class), value = @MethodSig("getChunk(I)Lnet/minecraft/world/level/chunk/LevelChunk;"), visibility = PUBLIC)
         public native LevelCube getChunk(int chunkIndex);
```

Comment:
I think in future we should try to have dasm annotations first, that way actual relevant annotations are are on the method
```suggestion
        @TransformFromMethod(owner = @Ref(ClientChunkCache.Storage.class), value = @MethodSig("getChunk(I)Lnet/minecraft/world/level/chunk/LevelChunk;"), visibility = PUBLIC)
        @Nullable public native LevelCube getChunk(int chunkIndex);
```

Reply:
agreed. and `@Nullable` specifically is a type annotation on the return type, so it should be directly before the return type (after modifiers)